Jobs.Fetcher.Twitter/Helpers/DbReader.cs

```diff
@@ -18,49 +18,98 @@ namespace Jobs.Fetcher.Twitter.Helpers {
 
     public static class DbReader {
 
-        public static User GetUserByUsername(string username, DataLakeTwitterDataContext dbContext) {
-            return dbContext.Users.FirstOrDefault(u => u.Username == username);
+        public static User GetUserByUsername(string username, DataLakeTwitterDataContext dbContext, Logger logger) {
+            try {
+                return dbContext.Users.FirstOrDefault(u => u.Username == username);
+            } catch (Exception e) {
+                logger.Error($"Error when looking for User {username} in the database");
+                logger.Verbose($"Error: {e}");
+                return null;
+            }
         }
 
-        public static IEnumerable<LocalAdsAccount> GetAdsAccounts(string username, DataLakeTwitterAdsContext dbContext) {
-            return dbContext.AdsAccounts.Where(a => a.Username == username).ToList();
+        public static IEnumerable<LocalAdsAccount> GetAdsAccounts(string username, DataLakeTwitterAdsContext dbContext, Logger logger) {
+            try {
+                return dbContext.AdsAccounts.Where(a => a.Username == username).ToList();
```

Comment:
Also here, I think this does not return an error when username is not found, it just returns an empty list